app/policies/article_policy.rb

```diff
@@ -144,13 +144,39 @@ def subscriptions?
     user_author? || user_super_admin?
   end
 
+  def elevated_user?
+    user_any_admin? || user_moderator?
+  end
+
   # this method performs the same checks that determine
-  # if the record can be featured
+  # if the record can be featured or if user can adjust
+  # any tag
   def revoke_publication?
     require_user!
     return false unless @record.published?
 
-    user_any_admin? || user_moderator?
+    elevated_user?
+  end
+
+  def allow_tag_adjustment?
+    require_user!
+
+    elevated_user? || tag_moderator_user?
+  end
+
+  def tag_moderator_user?
```

Comment:
@jeremyf I need help testing this method in `article_policy_spec`. In my mind, we would need to set up a number of scenarios and test that the correct boolean returns for each scenario. It's the scenarios setup that has me stumped.